### PR TITLE
Create checksum file for global tools

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

During refactoring of nuget.yml for SBOM changes, creating checksum for global tool packages was missed. This change adds back creation of a checksum file for global tool packages.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Comment by adityapatwardhan** [comment]
Test build to verify the new checksum file: https://dev.azure.com/mscodehub/PowerShellCore/_build/results?buildId=250887&view=logs&j=b68f2f44-6987-572d-9dbb-1c96660ee23d&t=815f2ee4-ed58-5f9d-3b75-921a3c1e1e34&l=35

**Comment by adityapatwardhan** [comment]
This change should be backported to 7.0, 7.1 and 7.2. It is being marked accordingly.

File: tools/releaseBuild/azureDevOps/templates/nuget.yml [hunk 1]
```diff
@@ -222,9 +222,36 @@ jobs:
       Write-Verbose -Verbose -Message $fileContent
 
       Copy-Item -Path "$packagePath\SHA512SUMS" -Destination '$(System.ArtifactsDirectory)\signed\' -verbose
-      Copy-Item -Path "$packagePath\SHA512SUMS" -Destination '$(System.ArtifactsDirectory)\signed\globaltool\' -verbose
     displayName: Generate checksum file
```
Comment (PaulHigin) [inline tools/releaseBuild/azureDevOps/templates/nuget.yml:225]:
```suggestion
    displayName: Generate checksum file for packages
```

**Review by PaulHigin — APPROVED** [review]
LGTM

**Review by TravisEz13 — APPROVED** [review]

File: tools/releaseBuild/azureDevOps/templates/nuget.yml (hunk 1 above)
Comment (daxian-dbw) [reply tools/releaseBuild/azureDevOps/templates/nuget.yml:225]: @adityapatwardhan Can you please address this comment? I can merge afterwards.

**Pull request merged**